app/src/main/java/com/pysiak/simple/clock/ui/clock/TimezonedbApi.java

```diff
@@ -0,0 +1,12 @@
+package com.pysiak.simple.clock.ui.clock;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface TimezonedbApi {
+
+    @GET("/v2.1/get-time-zone?format=json")
```

Comment:
Currently your request is broken ad doesn't work. The reason is that you haven't added "by=position" to the request. Fix it!